### PR TITLE
started new project for redesign

### PR DESCRIPTION
**Pull request by doctor3w**
Yup here is the base new one. Only SnapKit install. Go wild.

### CODE REVIEW

**Comment by doctor3w** [comment]
The best refactor yet

File: Recast/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        
+        window = UIWindow()
+        
+        let seriesViewControllerTest = SeriesViewController(nibName: nil, bundle: nil)
+        let nc = UINavigationController(rootViewController: seriesViewControllerTest)
```
Comment (mindylou) [inline Recast/AppDelegate.swift]: can you name this variable better? even though it's gonna be replaced later just for style conventions

File: old/Podcast/Keys.plist [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mindylou) [inline old/Podcast/Keys.plist]: aaaaa remove this!

**Review by mindylou — CHANGES_REQUESTED** [review]
Two quick changes otherwise it looks good!

**Comment by doctor3w** [comment]
All fixed!

**Pull request merged**